tests/rng_tools/tests/01-run.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2017 Bas Stottelaar <basstottelaar@gmail.com>
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import pexpect
+import sys
+import os
+
+
+def testfunc(child):
+    # RNG source
+    child.sendline("source 0")
+    child.sendline("seed 1337")
+
+    child.sendline("fips")
+    child.expect("Running FIPS 140-2 test, with seed 1337 using Tiny Mersenne Twister PRNG.")
+    child.expect("Monobit test: passed")
+    child.expect("Poker test: passed")
+    child.expect("Run test: passed")
+    child.expect("Longrun test: passed")
+
+    child.sendline("dump 10")
+    child.expect("1555530734")
+    child.expect("2178333451")
+    child.expect("2272913641")
+    child.expect("3790481823")
+    child.expect("3190025502")
+    child.expect("798555366")
+    child.expect("1918982324")
+    child.expect("1550167154")
+    child.expect("3454972398")
+    child.expect("1034066532")
+
+    child.sendline("entropy")
+    child.expect("Calculated 7.994709 bits of entropy from 10000 samples.")
```

Comment:
Weirdly exact number for a floating point computation. Can this be made into a comparison with tolerances? Some other math library may have different accuracy in the floating point functions used by the test which would cause this check to fail, even with exactly the same PRNG sequence.

Reply:
Changed this by having a regex on the last three digits.